drizzlepac/hapsequencer.py

```diff
@@ -484,9 +484,6 @@ def run_hap_processing(input_filename, diagnostic_mode=False, use_defaults_confi
             log.info("SVM Quality Assurance statistics have been requested for this dataset, {}.".format(input_filename))
             svm_qa.run_quality_analysis(total_obj_list, log_level=log_level)
 
-            # Compute WCS differences between the primary WCS and the alternates, except for OPUS WCS
-            svm_qa.report_wcs(total_obj_list)
-
         # Write out manifest file listing all products generated during processing
```

Comment:
Just noting the removal of 487-489 has already been checked into master with PR#668.

Reply:
Noted!